### ISSUE TITLE
Windows suggests an open file dialog every time I reload VS Code

### ISSUE THREAD

**Issue by mikbox74** [faelv/composer-companion#4]
Windows 7 x64, Path to the composer executable: `D:\\PHP\\composer.phar`:
![image](https://user-images.githubusercontent.com/1404546/75630082-a2d35f80-5bf8-11ea-9dee-0a1086cd1724.png)

**Comment by faelv** [comment]
Does this happens if you use ```composer.bat``` instead?
If you don't have a ```composer.bat``` file you can create one in the same folder as ```composer.phar```. Copy and paste the following content, then save:

```cmd
@echo OFF
setlocal DISABLEDELAYEDEXPANSION
"php.exe" "%~dp0composer.phar" %*
```
You may need to adjust the ```php.exe``` location.

**Comment by mikbox74** [comment]
Thank you for the tip, now the ext works properly! May I suggest you a PR with updated README containing this case?

**Comment by faelv** [comment]
Glad to hear it works now.
Yes, feel free to submit a PR if you want, I'll also add some more checks tomorrow to prevent this problem.

**Comment by faelv** [comment]
Closed by 2a195113b0bb91cc1eb874bdfbd0017e37d51d8a and #5
